app/aws/s3.py

```diff
@@ -96,13 +95,18 @@ def cleanup_old_s3_objects():
     time_limit = aware_utcnow() - datetime.timedelta(days=14)
     try:
         response = s3_client.list_objects_v2(Bucket=bucket_name)
-        print(f"RESPONSE = {response}")
         while True:
             for obj in response.get("Contents", []):
                 if obj["LastModified"] <= time_limit:
-                    current_app.logger.info(
-                        f"#delete-old-s3-objects Wanting to delete: {obj['LastModified']} {obj['Key']}"
-                    )
+
+                    try:
+                        remove_csv_object(obj["Key"])
+                        current_app.logger.info(
+                            f"#delete-old-s3-objects Deleted: {obj['LastModified']} {obj['Key']}"
```

Comment:
Thanks for still logging what's deleted here when it's successful, this way we have a record of the action and it'll still be possible to troubleshoot potential issues that come up with any missing files that we think _should_ be there!